src/Grid/Util/Trait/MetaData/LocalizedValueTrait.php

```diff
@@ -0,0 +1,40 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\StudioBackendBundle\Grid\Util\Trait\MetaData;
+
+use Pimcore\Bundle\StudioBackendBundle\Grid\Schema\Column;
+use Pimcore\Model\Asset;
+use Pimcore\Model\Element\ElementInterface;
+
+/**
+ * @internal
+ */
+trait LocalizedValueTrait
+{
+    private function getLocalizedValue(Column $column, ElementInterface $element): ?string
+    {
+        if (!$element instanceof Asset) {
+            return null;
```

Comment:
Just an Idea... Should we not throw an Exception here?

Reply:
Hmm, so you would need to call e.g. the object grid and put metadata.input or metadata.textarea as type. Yeah that is kinda a hard missconfiguration. We can throw an exception here too then. InvalidArgument should do it? WDYT?

Reply:
Done in https://github.com/pimcore/studio-backend-bundle/pull/310/commits/0d4ea9cb0f83c27af9ccd4546eb2a297a7b29c09